README.md

```diff
@@ -52,40 +55,95 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
+          <b style="margin-left:-20px">Tuner for NAS</b>
+          <li><a href="docs/en_US/Tuner/BuiltinTuner.md#NetworkMorphism">Network Morphism</a></li>
           <li><a href="examples/tuners/enas_nni/README.md">ENAS</a></li>
-          <li><a href="docs/en_US/BuiltinTuner.md#MetisTuner">Metis Tuner</a></li>
-          <li><a href="docs/en_US/BuiltinTuner.md#BOHB">BOHB</a></li>
-          <li><a href="docs/en_US/BuiltinTuner.md#GPTuner">GP Tuner</a></li>
         </ul>
-          <a href="docs/en_US/BuiltinAssessor.md">Assessor</a>
+          <a href="docs/en_US/Assessor/BuiltinAssessor.md">Assessor</a>
         <ul>
-          <li><a href="docs/en_US/BuiltinAssessor.md#Medianstop">Median Stop</a></li>
-          <li><a href="docs/en_US/BuiltinAssessor.md#Curvefitting">Curve Fitting</a></li>
+          <li><a href="docs/en_US/Assessor/BuiltinAssessor.md#Medianstop">Median Stop</a></li>
+          <li><a href="docs/en_US/Assessor/BuiltinAssessor.md#Curvefitting">Curve Fitting</a></li>
         </ul>
       </td>
       <td>
       <ul>
-        <li><a href="docs/en_US/LocalMode.md">Local Machine</a></li>
-        <li><a href="docs/en_US/RemoteMachineMode.md">Remote Servers</a></li>
-        <li><a href="docs/en_US/PaiMode.md">OpenPAI</a></li>
-        <li><a href="docs/en_US/KubeflowMode.md">Kubeflow</a></li>
-        <li><a href="docs/en_US/FrameworkControllerMode.md">FrameworkController on K8S (AKS etc.)</a></li>
+        <li><a href="docs/en_US/TrainingService/LocalMode.md">Local Machine</a></li>
+        <li><a href="docs/en_US/TrainingService/RemoteMachineMode.md">Remote Servers</a></li>
+        <li><b>Kubernetes based services</b></li>
+            <ul><li><a href="docs/en_US/TrainingService/PaiMode.md">OpenPAI</a></li>
+            <li><a href="docs/en_US/TrainingService/KubeflowMode.md">Kubeflow</a></li>
+            <li><a href="docs/en_US/TrainingService/FrameworkControllerMode.md">FrameworkController on K8S (AKS etc.)</a></li>
+            </ul>
       </ul>
       </td>
     </tr>
+    <tr align="center" valign="bottom">
+      <td style="border-top:#FF0000 solid 0px;">
+        <b>References</b>
+        <img src="docs/img/bar.png"/>
+      </td>
+      <td style="border-top:#FF0000 solid 0px;">
+        <b>References</b>
+        <img src="docs/img/bar.png"/>
+      </td>
+      <td style="border-top:#FF0000 solid 0px;">
+        <b>References</b>
+        <img src="docs/img/bar.png"/>
+      </td>
+    </tr>
+    <tr valign="top">
+     <td style="border-top:#FF0000 solid 0px;">
+      <ul>
+        <li><a href="docs/en_US/sdk_reference.rst">Python API</a></li>
+        <li><a href="docs/en_US/Tutorial/AnnotationSpec.md">NNI Annotation</a></li>
+        <li><a href="docs/en_US/TrialExample/Trials.md#nni-python-annotation">Annotation tutorial</a></li>
+        <li><a href="examples/trials/cifar10_pytorch/README.md">Example_cifar10_pytorch</a></li>
```

Comment:
They have doc link under "doc/en_US/TrialExample/".   I suggested that you change to this folder.